Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs*.deb gcmcore-linux*.deb packages*.deb
+
+echo "Installing dotnet SDK 3.1"
+
+sudo apt-get update -qq
+sudo apt-get install -y apt-transport-https && sudo apt-get update -qq
+
+sudo apt-get install -y lsb-release wget
+LSB_VERSION=$(lsb_release -rs | awk '/^[0-9]+\.[0-9]+$/ { print }')
+if [ -z "$LSB_VERSION" ]
+then
+	LSB_VERSION="20.04"
+fi
+wget https://packages.microsoft.com/config/ubuntu/"$LSB_VERSION"/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+
+if sudo apt-get install -y dotnet-sdk-3.1
```

Comment:
In the Azure Pipelines agent, we use the [install scripts](https://dotnet.microsoft.com/download/dotnet-core/scripts) to get .NET on the box. They've done some ~~heavy~~ medium lifting across various Linux flavors to get prereqs sorted and whatnot. This script looks pretty Ubuntu/Debian flavored so perhaps you 🤷‍♂️ but I thought I'd mention it.

Reply:
interesting! If we have any more problems with installing `dotnet` I will definitely replace these with something like `wget https://dotnet.microsoft.com/download/dotnet-core/scripts/v1/dotnet-install.sh && ./dotnet-install.sh`